### PR TITLE
fix(ui): Move legacy DSN to the top, Extend ClippedBox height

### PR DESCRIPTION
**Pull request by HazAT**
ref: https://github.com/getsentry/sentry-docs/pull/236

- Reordering fields to: `DSN` `DSN (Deprecated)` `rest ...`
- Make ClippedBox higher so it shows both DSN

client_keys

Fixes https://getsentry.atlassian.net/browse/SDK-214

### CODE REVIEW

File: src/sentry/static/sentry/app/views/settings/project/projectKeys/projectKeyCredentials.jsx [hunk 1]
```diff
@@ -47,6 +47,25 @@ class ProjectKeyCredentials extends React.Component {
 
     return (
       <React.Fragment>
+        {showDsn && (
+          <Field
+            label={t('DSN (Legacy)')}
```
Comment (mattrobenolt) [inline src/sentry/static/sentry/app/views/settings/project/projectKeys/projectKeyCredentials.jsx]: Should we maybe not call it Legacy? Legacy, to me, implies that I shouldn't use it and I'd look for the one that's not legacy anyways.

Comment (HazAT) [reply src/sentry/static/sentry/app/views/settings/project/projectKeys/projectKeyCredentials.jsx]: I've updated the wording for the help text maybe it's more clear now.

Also AFAIK we want that people move away from it and it should be a conscious choice if you still want to take it.

**Review by jan-auer — APPROVED** [review]

**Comment by benvinegar** [comment]
I'm not sure this should go at the top-top, but it shouldn't be buried below security header endpoints.

Can we move it below the "new" DSN? This would make it 2nd from top.

**Comment by HazAT** [comment]
@benvinegar This would also work.
I mean the good thing right now is, even if people copy the "Legacy" DSN to a new SDK it would still work BUT if they read and think about it for 1 second, they will read the text and decide/take a look what they really need.
Besides that, in the docs we everywhere highlight the "new" DSN soooo ... I think there is no perfect solution 🤷‍♂️

**Comment by mitsuhiko** [comment]
I think @billyvg should have a quick pass over this. Also unrelated but slightly related: we only show the legacy dsn currently on the 'error tracking' page on the top above the framework/lib selection.

File: src/sentry/static/sentry/app/views/settings/project/projectKeys/projectKeyCredentials.jsx [hunk 2]
```diff
@@ -47,6 +47,27 @@ class ProjectKeyCredentials extends React.Component {
...
+              'This DSN includes the secret which is no longer required by Sentry or newer versions of SDKs.'
+            )}
+            inline={false}
+            flexibleControlStateSize
+          >
+            <TextCopyInput>
+              {getDynamicText({
+                value: data.dsn.secret,
+                fixed: data.dsn.secret.replace(
+                  new RegExp(`\/${projectId}$`),
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/settings/project/projectKeys/projectKeyCredentials.jsx]: You missed a merge here

**Comment by billyvg** [comment]
Is our eventual goal to deprecate the legacy DSN? It does seem odd that legacy is at the top, but I think it’s fine that the most compatible one be there. 

However I think it’d be good to add help text explaining why to use the new DSN over the legacy one. Because I honestly don’t why know why and would end up using the top most item.

**Comment by jan-auer** [comment]
The Legacy DSN **is** deprecated already, and only needed for old SDK versions / on-premise customers. The new DSN was formerly called "Public DSN".

**Comment by billyvg** [comment]
In that case, why not use "deprecated" instead of "legacy" and move it under `DSN`

File: src/sentry/static/sentry/app/views/settings/project/projectKeys/projectKeyCredentials.jsx (hunk 2 above)
Comment (HazAT) [reply src/sentry/static/sentry/app/views/settings/project/projectKeys/projectKeyCredentials.jsx]: Sorry, not really sure what you mean by that 🙃

**Comment by HazAT** [comment]
I've moved `DSN (Deprecated)` to the second place, updated the first comment to reflect the current state.
I think it's ready to merge 👍

**Comment by billyvg** [comment]
Check out the diff, it’s been changed to make sure the dsn is stable in
tests.

**Comment by HazAT** [comment]
@billyvg Got it! Should be fine now.

**Review by billyvg — APPROVED** [review]

**Pull request merged**